README.rst

```diff
@@ -1,11 +1,12 @@
 pycsob
 ======
 
-.. image:: https://circleci.com/gh/TwistoPayments/pycsob.svg?style=svg
-   :target: https://circleci.com/gh/TwistoPayments/pycsob
+.. image:: https://travis-ci.org/druids/pycsob.svg?branch=master
+    :target: https://travis-ci.org/druids/pycsob
+
+.. image:: https://codecov.io/gh/druids/pycsob/branch/master/graph/badge.svg
+    :target: https://codecov.io/gh/druids/pycsob
```

Comment:
I think we don't have code coverage set up. The link is not working.

Reply:
Yes, it is not working. Is it possible that it be set up?

Reply:
I've never used codecov, only coveralls.

Reply:
Please either make it working (or try coveralls) or remove it.

Reply:
Done.